src/rpm/package.rs

```diff
@@ -819,7 +821,7 @@ impl RPMPackageMetadata {
                             mode: mode.into(),
                             modified_at: utc.timestamp_opt(mtime as i64, 0u32).unwrap(), // shouldn't fail as we are using 0 nanoseconds
                             digest,
-                            category: FileCategory::from_u32(flags).unwrap_or_default(),
+                            flags: FileFlags::from_bits(flags).expect("Unexpected rpm file flags found"),
```

Comment:
Should be an error, since there it's within a `try_fold` anyways, `panic` seems unnecessary.